--- electric-6.08.orig/Makefile.in	2003-05-22 20:35:22.000000000 +0200
+++ electric-6.08/Makefile.in	2003-11-03 10:45:47.000000000 +0100
@@ -61,8 +61,8 @@
 
 ########################################################### TCL INTERPRETER
 ##### If building with a TCL interpreter, uncomment these two lines
-# LANGTCLCFLAGS =  -DFORCETCL=1
-# LANGTCLLIBS = -ltcl
+LANGTCLCFLAGS =  -DFORCETCL=1
+LANGTCLLIBS = -ltcl
 ###########################################################
 
 
