diff -durN electric-6.01.orig/Makefile.in electric-6.01/Makefile.in
--- electric-6.01.orig/Makefile.in	Sat Jul 15 17:29:24 2000
+++ electric-6.01/Makefile.in	Sun Oct 22 10:14:52 2000
@@ -31,7 +31,8 @@
 ############################################### TCL INTERPRETER ###############
 ##### If building with a TCL interpreter, uncomment the next two lines
 # LANGTCL_OBJS= $(TCL_OBJS)
-# LANGTCLCFLAGS =  -DFORCETCL=1 -Isrc/tcl/generic
+LANGTCLLIBS = -ltcl
+LANGTCLCFLAGS =  -DFORCETCL=1
 ###############################################################################
 
 
@@ -53,7 +54,7 @@
 # LANGJAVA_MACHINE = i386
 # LANGJAVA_OS = linux
 # LANGJAVA_THREADLIB = -lhpi
-###############################################################################
+###############################################################################NG
 
 
 ############################################### MISCELLANEOUS EXTRAS ##########
@@ -390,7 +391,7 @@
 EXTRA_CFLAGS = $(LANGLISPCFLAGS) $(LANGTCLCFLAGS) $(LANGJAVACFLAGS) $(CADENCECFLAGS)
 CFLAGS = $(EXTRA_CFLAGS) $(DEBUG) -I$(HDS)
 XCFLAGS = @X_CFLAGS@ $(CFLAGS) $(TIFFLAGS) $(XPOWER)
-EXTRA_LIBS = $(LANGJAVALIBS)
+EXTRA_LIBS = $(LANGJAVALIBS) $(LANGTCLLIBS)
 LIBS = @X_PRE_LIBS@ @X_LIBS@ $(XTLIBS) -lX11 -lm @X_EXTRA_LIBS@ $(EXTRA_LIBS)
 INSTALL = cp -r
 INSTALL_PROGRAM = $(INSTALL)
